packages/react-native-sdk/src/utils/push/config.ts

```diff
@@ -0,0 +1,11 @@
+import { StreamVideoRN } from '../StreamVideoRN';
+
+export const getPushConfig = () => {
+  const pushConfig = StreamVideoRN.getConfig().push;
+  if (pushConfig === undefined) {
+    throw Error(
+      'No config for push notifications have been set in StreamVideoRN.config. So push notifications cannot be enabled for this app',
+    );
```

Comment:
Discussion: what if I have a meeting app and I don't want to set PN?
Don't get me wrong, I like the idea, that people won't be able to miss that